DMTN-222.tex

```diff
@@ -313,23 +323,64 @@ \subsection{Distribution}
   \hline
 \end{tabular}
 
-\subsubsection{Summit Cleanup}
+\subsubsection{Distribution With Rucio}
+
+The way to copy calibration files from \texttt{\/repo\/main} to the Summit, BTS, and the other DFs is via \texttt{rucio}.
+Note that \texttt{rucio} does not have access to the \texttt{embargo} repo.
+
+Using \texttt{rucio} involves first registering the datasets with the \texttt{rucio} service, and then creating a ``rule'' which says where the files should be synchronized.
+Once the rule is created then the \texttt{rucio} service will do the necessary synchronizations, and will place the calibrations in special S3 calibration buckets at BTS and the Summit and any other DFs that are configured.
+The S3 calibration buckets are configured such that only \texttt{rucio} has write access, and these files cannot be directly deleted (on purpose or accident).
+We have not yet figured out the plans for removing old calibrations.
+
+At the current time \texttt{rucio} will only synchronize the files, and it is the responsibility of the person doing the deployment to copy over the above \texttt{export.yaml} and do the ingestion and the final collection chaining.
+In the near future we expect the ingestion will be done automatically, though explicit deployment chaining is going to be a manual task for the foreseeable future.
+
+The following is how a set of flats can be synchronized.
+Note that the \texttt{DSLABEL} can be anything.
+But using the ticket name is very important to ensure that the dataset name is unique for each set of registered calibrations.
+
+\begin{verbatim}
+
+DSLABEL=flat
+
+COLLECTION=$INSTRUMENT/calib/$TICKET
+DATASET=Dataset/$INSTRUMENT/$DSLABEL/$TICKET
+
+for dstype in flat; do
+  rucio-register data-products \
+      -s 10 \
+      -C /sdf/data/rubin/shared/calibration_archive/rucio/main-calib-config.yaml \
+      -r /repo/main \
+      -t $dstype \
+      -c $COLLECTION \
+      -d $DATASET
+done
+
+rucio did update --close --did ancillary:$DATASET
+
+rucio did content list --did ancillary:$DATASET
+
+RULEID=$(rucio rule add \
+    --rses 'SLAC_BUTLER_DISK|BASE_CALIB_DISK|SUMMIT_CALIB_DISK' \
+    --did ancillary:$DATASET \
+    --copies 3)
+rucio rule show --rule $RULEID
+
+rucio replica list dataset --did ancillary:$DATASET
```

Comment:
And in doing so, we can explicitly state that this will need to be repeated until the result on L374 happens.